report/README.txt

```diff
@@ -0,0 +1 @@
+// This directory is intended to hold the junit XML reports generated by fv and unit tests.
```

Comment:
We don't normally check in empty dirs, why not have the makefile create this as needed?

Reply:
Its referred to in the tests themselves.  If someone were to run the test manually (without the makefile), the directory wouldn't exist.  Not sure what would happen then...